src/components/targets/createTarget/form.tsx

```diff
@@ -20,6 +21,17 @@ const OPTIONS: SelectDropDownProps[] = [
 ];
 
 const Form = () => {
+
+  const [title, setTitle] = useState("");
+  const [arealength, setArealength] = useState("");
+  const [topic, setTopic] = useState("");
+
+  const createTargetRes = async (title: String, arealength: String, topic: String) => {
```

Comment:
beware of using the `string` instead of `String` (notice the capital letter)
```suggestion
  const createTargetRes = async (title: string, arealength: string, topic: string) => {
```